lib/WTSI/NPG/HTS/PacBio/Sequel/APIClient.pm

```diff
@@ -222,7 +281,7 @@ sub _get_content{
   my($self, $query) = @_;
 
   $self->debug("Getting query URI '$query'");
-  my $ua = LWP::UserAgent->new;
```

Comment:
Perhaps this could be set in the constructor (and if not supplied, create new one)? It would be more explicit and would mean that the tests wouldn't need to override the private `_get_user_agent`.

Reply:
Yes bad Carol. Changed.